installation.md

```diff
@@ -152,11 +152,55 @@ with `M-x company-mode`.
 - **Debug logs**: available in the `EGLOT events` buffer.
 - **Command-line flags and alternate binary**: instead of adding `"clangd"`
   to `eglot-server-programs`, add `("/path/to/clangd" "-log=verbose")` etc.
+
+---
+
+[lsp-mode](https://github.com/emacs-lsp/lsp-mode) can also be configured to work with clangd. It has support for all standard clangd features and some of the extensions, like completion scores, or type hierarchies.
+
+Install lsp-mode with `M-x package-install RET lsp-mode RET`.
+
+Add the following to `~/.emacs` to enable lsp-mode:
+
+```
+(require 'lsp-mode)
+(add-hook 'c-mode-hook #'lsp)
+(add-hook 'c++-mode-hook #'lsp)
+```
+
+### Indexing status
+
+lsp-mode can process indexing notifications from clangd and show you progress in the mode line.
+
+![Indexing status in lsp-mode](screenshots/lsp_mode_indexing.png)
+
+### Clang-tidy error explanations
```

Comment:
These *are* worth mentioning as there's some integrations with other plugins/commands that users need to know.

But having full headings and screenshots for each is too much weight, as these are all relatively minor features (e.g. for the other editors we just show a single screenshot with completion and/or diagnostics, which are probably the most prominent features).

I'd suggest adding a bullet list instead:

```
### Integration with other packages:

 - See documentation for clang-tidy linting issues with `M-x flycheck-explain-error-at-point`.
 - Search for workspace symbols using [lsp-ivy] or [helm-lsp]
 - [lsp-ui] provides extra UI features: ...
```

Reply:
Good point. I've added a summary of the integrations `lsp-mode` has with other packages.